examples/3d/query_gltf_primitives.rs

```diff
@@ -0,0 +1,96 @@
+//! Query primitives in a glTF scene.
+
+use bevy::{
+    gltf::GltfMaterialName,
+    pbr::{CascadeShadowConfigBuilder, DirectionalLightShadowMap},
+    prelude::*,
+    render::mesh::VertexAttributeValues,
+};
+use rand::{random, Rng};
+
+fn main() {
+    App::new()
+        .insert_resource(DirectionalLightShadowMap { size: 4096 })
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .add_systems(Update, random_color_and_position)
+        .run();
+}
+
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    commands.spawn((
+        Camera3dBundle {
+            transform: Transform::from_xyz(0.6, 1.6, 11.3)
+                .looking_at(Vec3::new(0.0, 0.0, 3.0), Vec3::Y),
+            ..default()
+        },
+        EnvironmentMapLight {
+            diffuse_map: asset_server.load("environment_maps/pisa_diffuse_rgb9e5_zstd.ktx2"),
+            specular_map: asset_server.load("environment_maps/pisa_specular_rgb9e5_zstd.ktx2"),
+            intensity: 500.0,
+        },
+    ));
+
+    commands.spawn(DirectionalLightBundle {
+        directional_light: DirectionalLight {
+            shadows_enabled: true,
+            ..default()
+        },
+        // This is a relatively small scene, so use tighter shadow
+        // cascade bounds than the default for better quality.
+        // We also adjusted the shadow map to be larger since we're
+        // only using a single cascade.
+        cascade_shadow_config: CascadeShadowConfigBuilder {
+            num_cascades: 1,
+            maximum_distance: 1.6,
+            ..default()
+        }
+        .into(),
+        ..default()
+    });
+    commands.spawn(SceneBundle {
+        scene: asset_server.load(
+            GltfAssetLabel::Scene(0).from_asset("models/GltfPrimitives/gltf_primitives.glb#Scene0"),
```

Comment:
```suggestion
            GltfAssetLabel::Scene(0).from_asset("models/GltfPrimitives/gltf_primitives.glb"),
```
That should be redundant.